### DISCUSSION TITLE
P-Steps to support TranslateService in @ngx-translate/core

### DISCUSSION THREAD

**Discussion by pctimhk** [primefaces/community#1036] [PrimeNG] [answered]
I would like to develop mulit-lanuage web portal. However, P-Steps did not support change language event for TranslateService in @ngx-translate/core. Please support TranslateService if possible.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we will wait for the community interaction to improve this feature.
